sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py

```diff
@@ -40,3 +40,4 @@ class ScheduleSchema(ResourceSchema):
     creation_context = NestedField(CreationContextSchema, dump_only=True)
     is_enabled = fields.Boolean(dump_only=True)
     provisioning_state = fields.Str(dump_only=True)
+    properties = fields.Dict(keys=fields.Str(), values=fields.Str(allow_none=True))
```

Comment:
Do we support nested dict in properties?

Reply:
I remember that's not supported

Reply:
Nested dict in sdk will become string after submission, while nested dict in cli is not supported and will raise validation error, user should write string manually.
![image](https://user-images.githubusercontent.com/24237253/216534996-643505a2-ce91-4e0c-8819-7af34eeaf4e6.png)
![image](https://user-images.githubusercontent.com/24237253/216535049-ea15c6d7-4e86-42ec-8937-a96a6bbf2c94.png)

Reply:
seems good for now. Let's revisit in future.